asllib/Parser.mly

```diff
@@ -463,9 +463,9 @@ let stmt ==
       | RETURN; ~=ioption(expr);                             < S_Return >
       | x=IDENTIFIER; args=plist(expr); ~=nargs;             < S_Call   >
       | ASSERT; e=expr;                                      < S_Assert >
-      | DEBUG; e=expr;                                       < S_Debug >
-      | le=lexpr; EQ; e=expr;
-          {  S_Assign (le,e,V1) }
+      | PRINT; args=plist(expr);                             { S_Print { args; debug = false } }
+      | DEBUG; args=clist(expr);                             { S_Print { args; debug = true } }
```

Comment:
Why not use plist to parse the arguments of DEBUG? Syntax `DEBUG x,y` is rather weird.